contracts/FractalUsul.sol

```diff
@@ -0,0 +1,74 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+// Usul was previously named "Seele" and SekerDAO was TokenWalk
+// that's where this naming differences are coming from
+import "@tokenwalk/seele/contracts/Usul.sol";
+
+abstract contract FractalUsul is Usul {
+  struct Transaction {
+    address to;
+    uint256 value;
+    bytes data;
+    Enum.Operation operation;
+  }
+  
+  struct Proposal {
+    bool canceled;
+    uint256 timeLockPeriod; // queue period for safety
+    bytes32[] txHashes;
+    Transaction[] transactions;
```

Comment:
Are we going to store the transactions on the proposal? In order to save gas / TX costs, we discussed potentially just emitting the transactions data in an event, so that it could be retrieved off-chain later (and then submitted to execute).